File: Primeira lista pt2/Exercicio05.java
public class Exercicio05 {
    public static void main(String[] args) {

        double carro1 = 10;
        double carro2 = 20;

        int volta = 0;

        while (carro1 <= carro2) {

            volta++;

            carro1 += 3;
            carro2 += 1.5;

            System.out.printf(
                "Volta %d - Carro 1: %.1f km/h | Carro 2: %.1f km/h%n",
                volta, carro1, carro2
            );
        }

        System.out.println("Total de voltas: " + volta);
    }
}

File: lista pt2/Exercicio05.js
let carro1 = 10;
let carro2 = 20;

let volta = 0;

while (carro1 <= carro2) {

    volta++;

    carro1 = carro1 + 3;
    carro2 = carro2 + 1.5;

    console.log(
        "Volta " + volta +
        " - Carro 1: " + carro1 + " km/h" +
        " | Carro 2: " + carro2 + " km/h"
    );
}

console.log("Total de voltas: " + volta);